riscos_toolbox/application.py

```diff
@@ -1,15 +1,48 @@
-from .events import EventHandler
+from .events import EventHandler, registered_wimp_events
+from consts import Wimp
```

Comment:
You named the module 'riscos_toolbox/_consts.py' but you're importing `consts` here. Did I miss something about how these are imported?

Reply:
No. It’s a typo. Or rather I ended up merging it with the file from the other PR and for the names muddled.